com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -342,6 +347,12 @@ void Update()
             var filename = "No Asset";
             if (controller != null)
             {
+                if (m_HasChanged)
+                {
+                    m_HasChanged = false;
+                    controller.graph.SanitizeForImport();
```

Comment:
The cost of this call probably depends on the size/complexity of the graph but with the default template it's 0ms.

Reply:
I have to check twice, isn't the SanitizeForImport already called [here](https://github.com/Unity-Technologies/Graphics/blob/171f783523aa4405adc946f18990a0b11a378a62/com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs#L38) ? The sanitized state isn't serialized, so, I would expect to be false while trying to reload but maybe I'm wrong. 🤔 
The problem of calling `SanitizeForImport();` here is it will hide some issue when graph is opened.

Reply:
That's correct, I'll see why the call to Sanitize in postproceessor is not enough (or is not called)

Reply:
In the end, should we postponed this PR ? The change here seems really risky (regarding dandling import we have right now).

Reply:
Yes, you're right.
That could be a test case for future importer refactorization.